docs/use-linea/bridge-funds/use-etherscan.md

```diff
@@ -0,0 +1,107 @@
+1. Navigate to Contract > Write as Proxy tab
+1. Connect your wallet by clicking on the “Connect with web3” button
+1. Open the `BridgeToken` function form ![bridge token](../../assets/etherscan-bridge/bridge-erc20-2.png)
+1. Complete it with the following information:
+
+<table>
+    <tr>
+        <td align="left"><b>BridgeToken (payableAmount)</b></td>
+        <td align="left">This is the fee paid to the postman message service to execute the transfer on the second layer automatically. Users will eventually receive _fee. We recommend providing a 0.001 ETH fee to ensure proper execution. Note: the amount is in wei, so 0.001 ETH = 1000000000000000.
+      </td>
+    </tr>
+    <tr>
+        <td align="left"><b>_token</b></td>
+        <td align="left">This is the address of the ERC20 to bridge</td>
+    </tr>
+    <tr>
+        <td align="left"><b>_amount</b></td>
+        <td align="left">Note: the amount is in the lowest decimal. To send 1 $UNI which has 18 decimals, you need to input 1000000000000000000</td>
+    </tr>
+    <tr>
+        <td align="left"><b>_recipient</b></td>
+        <td align="left">This is the recipient address on the other layer</td>
+    </tr>
+</table>
+
+1. Validate the transaction on your wallet and you’re done 🎉! Your funds should arrive on the other layer under a few minutes (L2 -> L1 bridging is a bit longer and can take up to ~15min)
+
+## Bridge ERC20 from Linea to Goerli
+
+Follow the same steps as above, except use the [Token Bridge transparent proxy address (0xB191E3d98074f92584E5205B99c3F17fB2068927)](https://goerli.lineascan.build/address/0xB191E3d98074f92584E5205B99c3F17fB2068927#writeProxyContract) as spender on Linea, and use the [Goerli ERC20 Token Bridge transparent proxy contract](https://goerli.etherscan.io/address/0xaA012D038E6440535Ec66eDf2DA592F4F8398133#writeProxyContract) to bridge the token.
+
+## Manually complete a bridge transaction
+
+:::note This step is not mandatory unless you didn’t send enough fees when initiating your bridge transaction :::
+
+1. Browse your `Transaction Details` page of the transaction you sent on the source layer
+1. Go to the `Logs` tab
+1. Find the log for the `MessageSent` event and keep it open for a later step ![message sent](../../assets/etherscan-bridge/logs.png)
+1. Go to the [Message Service contract](../../developers/bridge-architecture/message-service.mdx#contracts) on the other Layer ([0xC499a572640B64eA1C8c194c43Bc3E19940719dC](https://goerli.lineascan.build/address/0xC499a572640B64eA1C8c194c43Bc3E19940719dC) on L2)
+1. Navigate to the `Write Proxy` tab, connect your wallet, and call the `claimMessage` function using the parameters found in the logs MessageSent checked earlier. Note: `bytes` argument need to start with `0x0` ![write claimMessage](../../assets/etherscan-bridge/claimmessage.png)
+1. Confirm the transaction on your wallet
```

Comment:
All the paragraph is included in the note above. I don't know if it's intended

Reply:
? I don't understand this comment

Reply:
whoops that was weird autoformatting